locations/spiders/stc_se.py

```diff
@@ -0,0 +1,21 @@
+import scrapy
+
+from locations.categories import Categories, apply_category
+from locations.dict_parser import DictParser
+
+
+# Sats (Gym chain in the nordics) is active in SE, NO, DK, FI.
+# This spider is scraping the gym centers in all countries.
+class StcSESpider(scrapy.Spider):
+    name = "stc_se"
+    item_attributes = {"brand": "STC", "brand_wikidata": "Q124061743"}
+    start_urls = ["https://www.stc.se/assets/clubs.json"]
+
+    def parse(self, response):
+        for club in response.json():
+            item = DictParser.parse(club)
+            item["branch"] = item["name"]
+            item["name"] = "STC"
```

Comment:
We don't specify our own name in spiders. If the page doesn't have an explicit name, we shouldn't set it.

Reply:
I'm just moving what the DictParser identifies as the name into the "branch" attribute.
The API returns the branch name in the "name" attribute but the branch name is not really visible on the outside of these gyms, on the outside all are branded with a big "STC" sign and the branch name is often only the street name.

Reply:
Please remove the `item["name"] = "STC"` line.

Reply:
But it will have a bad value. We can pop it, then let NSI give us the name.

```suggestion
            item["branch"] = item.pop("name")
```